languages/csharp/Bitwarden.Sdk/Bitwarden.Sdk.csproj

```diff
@@ -0,0 +1,71 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+  <TargetFramework>net6.0</TargetFramework>
+  <PackageReadmeFile>README.md</PackageReadmeFile>
+  <ImplicitUsings>enable</ImplicitUsings>
+  <Nullable>enable</Nullable>
+  <AssemblyName>BitwardenSdk</AssemblyName>
+  <PackageId>BitwardenSdk</PackageId>
+  <Title>Bitwarden Sdk</Title>
+  <Authors>Bitwarden Inc.</Authors>
+  <Description>Bitwarden Sdk</Description>
+  <Copyright>Bitwarden Inc.</Copyright>
+  <RepositoryUrl>https://github.com/bitwarden/sdk/tree/master/languages/csharp</RepositoryUrl>
+  <PackageTags>Bitwarden;Sdk;.NET</PackageTags>
+  <Product>BitwardenSdk</Product>
+  <PackageIcon>bitwarden.png</PackageIcon>
+  <RepositoryType>Git</RepositoryType>
+  <RootNamespace>Bitwarden.Sdk</RootNamespace>
+</PropertyGroup>
+
+<ItemGroup>
+  <PackageReference Include="System.Text.Json" Version="7.0.3" />
+  <None Include="bitwarden.png" Pack="true" PackagePath="\" />
+  <None Update="README.md" Pack="true" PackagePath="\" />
```

Comment:
We probably also want to package a copy of the license.

Reply:
I can copy it in the root od c# code

Reply:
done